### PR TITLE
Work with local libhoclient library and local host orchestrator api/v1 package in e2etests.

### PR DESCRIPTION
**Pull request by ser-io**

### CODE REVIEW

File: frontend/src/libhoclient/host_orchestrator_client.go [hunk 1]
```diff
@@ -386,6 +398,55 @@ func (c *HostOrchestratorServiceImpl) ListCVDs() ([]*hoapi.CVD, error) {
 	return res.CVDs, nil
 }
 
+func (c *HostOrchestratorServiceImpl) Powerwash(group, name string) error {
```
Comment (jemoreira) [inline frontend/src/libhoclient/host_orchestrator_client.go:401]: nit: Rename `group, name` as `group, instance` or `groupName, instanceName`.

File: frontend/src/libhoclient/host_orchestrator_client.go
Comment (jemoreira) [inline frontend/src/libhoclient/host_orchestrator_client.go:1]: These changes look like they should be in a different PR

**Review by jemoreira — APPROVED** [review]

File: frontend/src/libhoclient/host_orchestrator_client.go
Comment (ser-io) [reply frontend/src/libhoclient/host_orchestrator_client.go:1]: done.

File: frontend/src/libhoclient/host_orchestrator_client.go (hunk 1 above)
Comment (ser-io) [reply frontend/src/libhoclient/host_orchestrator_client.go]: this code was removed from this PR following the comment below. I'll address the comment in the following PR containing this change.
